src/System/Taffybar/Widget/Brightness.hs

```diff
@@ -0,0 +1,13 @@
+module System.Taffybar.Widget.Brightness (brightnessBarNew) where
```

Comment:
I've started adding imports for all widgets in System.Taffybar.Widget, so we should probably do that with this.